dev/archetypes/maven-java/src/main/resources/archetype-resources/__service1Name__-impl/src/main/resources/application.conf

```diff
@@ -19,19 +19,15 @@ cassandra-journal.keyspace = ${${service1Name}.cassandra.keyspace}
 cassandra-snapshot-store.keyspace = ${${service1Name}.cassandra.keyspace}
 lagom.persistence.read-side.cassandra.keyspace = ${${service1Name}.cassandra.keyspace}
 
-# -----------------------------------------------------------------
-# DON'T REMOVE!!!
-# The properties below overrides the Lagom defaults.
+# The properties below override Lagom default configuration with the recommended values for new projects.
 #
-# For compatibility reasons we are not yet changing the defaults,
-# but for new projects, we highly recommend the values below.
-# Future Lagom releases will have these values by default.
+# Lagom has not yet made these settings the defaults for backward-compatibility reasons.
 
-# prefer 'ddata' over 'persistence' for new projects
+# Prefer 'ddata' over 'persistence' to share cluster sharding state for new projects.
+# See https://doc.akka.io/docs/akka/current/cluster-sharding.html#distributed-data-vs-persistence-mode
 akka.cluster.sharding.state-store-mode = ddata
 
-# enable new Akka serializer for akka.Done (since akka 2.5.8)
+# Enable the serializer for akka.Done provided in Akka 2.5.8+ to avoid the use of Java serialization.
 akka.actor.serialization-bindings {
   "akka.Done" = akka-misc
 }
-# -----------------------------------------------------------------
```

Comment:
Just a few comments on the reasoning for my rewording these comments:

- We don't want to freak people out too much with all caps and exclamation points :smile:
- It's best if users can understand the reason for these configurations and read references when available
- I wanted to avoid using "we" because these files become part of users' projects and evolve from there, so it won't necessarily be obvious to everyone reading that "we" means "the Lagom team".
- I removed the "Future Lagom releases..." part because it's not that helpful to the reader. Which future Lagom release? What can I do about it? We don't know so it's just clutter.